### PR TITLE
Fixerthefix

### PR DESCRIPTION
**Pull request by Pomax**
fixes touch event breakage in browsers without touch events, and adds a trash icon when you have an element selected. Also centers elements when adding new elements, to address https://github.com/mozilla/webmaker-app/issues/1631

### CODE REVIEW

File: www_src/pages/project/project.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ var Project = React.createClass({
           <div className="inner">
             <div className="positionables">{ positionables }</div>
           </div>
-        </div>
+        </div>3
```
Comment (k88hudson) [inline www_src/pages/project/project.jsx]: this might be unintentional?

**Comment by thisandagain** [comment]
:+1: Thanks for the quick turnaround on this! :smile:

File: www_src/pages/project/project.less [hunk 2]
```diff
@@ -154,6 +154,16 @@
           transform: translateY(0);
         }
       }
+
+      &.delete {
+        position: fixed;
+        top: 3em;
+        left: 3em;
+        z-index:12345678;
```
Comment (gvn) [inline www_src/pages/project/project.less]: :(

File: www_src/pages/project/project.jsx [hunk 3]
```diff
@@ -65,13 +68,20 @@ var Project = React.createClass({
...
+  componentDidMount: function() {
+    this.dims = this.refs.container.getDOMNode().getBoundingClientRect();
+  },
+
   toggleAddMenu: function () {
     this.setState({showAddMenu: !this.state.showAddMenu});
   },
 
   formPositionables: function(content) {
     return content.map((m, i) => {
```
Comment (gvn) [inline www_src/pages/project/project.jsx]: Could you use more semantic var names?

File: www_src/pages/project/project.less (hunk 2 above)
Comment (Pomax) [inline www_src/pages/project/project.less]: haha, riiiight let me change that back to a real value =D

**Comment by gvn** [comment]
LGTM. We should fix the trash button location in a follow up ticket...

**Comment by Pomax** [comment]
followup for button: https://github.com/mozilla/webmaker-app/issues/1653

**Pull request merged**